src/Pages/main/Main.js

```diff
@@ -1,8 +1,40 @@
 import React, { Component } from "react";
+import Nav from '../../Components/nav/Nav';
+import Footer from "../../Components/footer/Footer";
+import LinkBox from "../../Components/linkBox/LinkBox";
+import Account from '../account/Account';
+import Register from '../register/Register';
+import './Main.scss';
 
 class Main extends Component {
+  state = {
+    scrollPosition: 0,
+    scrollDirection: true
+  };
+
+  componentDidMount(){
+    window.addEventListener('scroll', (e) => {
+      this.setState({scrollPosition: e.srcElement.scrollingElement.scrollTop});
+    });
+  };
+
   render() {
-    return <div></div>;
+    return (
+      <div className='main'>
+        <Nav scrollPosition={this.state.scrollPosition}/>
+        <div className='mainBG1'>
+          <Account />
+          {/* <Register /> */}
+        </div>
+        <div className='mainBG2'></div>
+        <div className='mainBG3'></div>
+        <div className='mainBG4'></div>
+        <div className='mainBGShort'></div>
+        <div className='mainBG5'></div>
```

Comment:
```suggestion
        <div className="mainBG2" />
        <div className="mainBG3" />
        <div className="mainBG4" />
        <div className="mainBGShort" />
        <div className="mainBG5" />
```
1. 셀프 클로징 태그 써주세요
2. 클래스명 등에 숫자를 사용하는 건 좋지 못한 습관입니다. 보다 구체적인 이름을 지어주세요.